UnityMcpBridge/UnityMcpServer~/src/tools/read_console.py

```
@@ -1,44 +1,34 @@
+        count: Annotated[int, "Max messages to return"] | None = None,
+        filter_text: Annotated[str, "Text filter for messages"] | None = None,
+        since_timestamp: Annotated[str,
+                                   "Get messages after this timestamp (ISO 8601)"] | None = None,
+        format: Annotated[Literal['plain', 'detailed',
+                                  'json'], "Output format"] | None = None,
+        include_stacktrace: Annotated[bool,
+                                      "Include stack traces in output"] | None = None
+    ) -> dict[str, Any]:
+        ctx.info(f"Processing read_console: {action}")
         # Get the connection instance
         bridge = get_unity_connection()
```

Ruff (0.13.1)

Local variable `bridge` is assigned to but never used

Remove assignment to unused variable `bridge`

(F841)